sonar-project.properties

```diff
@@ -1 +1 @@
-sonar.exclusions=test/minimaltemplate/**/*
\ No newline at end of file
+sonar.exclusions=test/minimaltemplate/**/*,test/commands/diff.test.ts
```

Comment:
yeah, I vote for disabling sonar for all tests by default, it is causing confusion only

```suggestion
sonar.exclusions=test/**/*
```

Reply:
This didn't disable sonar :(